lib/main_common.pm

```diff
@@ -1773,7 +1773,8 @@ sub load_rollback_tests {
     return if check_var('ARCH', 's390x');
     # On Xen PV we don't have GRUB.
     # For continuous migration test from SLE11SP4, the filesystem is 'ext3' and btrfs snapshot is not supported.
-    loadtest "boot/grub_test_snapshot" unless check_var('VIRSH_VMM_TYPE', 'linux') || check_var('FILESYSTEM', 'ext3');
+    # For HPC migration test with 'management server' role, the filesystem is 'xfs', btrfs snapshot is not supported.
+    loadtest "boot/grub_test_snapshot" unless check_var('VIRSH_VMM_TYPE', 'linux') || check_var('FILESYSTEM', 'ext3') || check_var('FILESYSTEM', 'xfs');
```

Comment:
As alternative we could replace last to conditions with `get_var('FILESYSTEM') =~ /ext3|xfs/`

Reply:
This looks better. Thanks.

Reply:
Verify log: http://openqa.nue.suse.com/tests/3909170